frontend/src/app/about/page.tsx

```diff
@@ -44,19 +44,37 @@ export default function About() {
         }}
       >
         <Typography variant="h1" component="h1" gutterBottom>
-          About Us
+          Our Story
         </Typography>
         <Typography variant="body1" component="p">
-          As two Asian American women, we understand how difficult it is to find wedding artists who know how to highlight your natural beauty.
-          A makeup artist who is an experienced professional is not necessarily experienced with Asian features or styles of makeup.
+          As Asian Americans, we understand how difficult it is to find wedding artists who know how to highlight your natural beauty. 
         </Typography>
         <Typography variant="body1" component="p">
-          We&apos;re building this directory to help Asian couples find artists who are recommended within their local communities.
-          We&apos;re asking our friends and family, scouring the web, scrolling Instagram, and looking through Reddit and Facebook, so you don&apos;t have to.
-          The artists here are people who other couples have recommended, and we&apos;re excited to share their recommendations with you.
+          When Katrina was planning her own wedding in Boston, she struggled to find makeup artists experienced with Asian features.
```

Comment:
```suggestion
          When Katrina was planning her own wedding, she struggled to find makeup artists experienced with Asian features.
```

Reply:
hmm I was thinking that Boston is useful context because we've seen more demand on the east coast!